sympy/integrals/__init__.py

```diff
@@ -9,7 +9,11 @@
     >>> integrate(sin(x),x)
     -cos(x)
 """
+from .deltafunctions import deltaintegrate
+from .heurisch import heurisch, heurisch_wrapper
 from .integrals import integrate, Integral, line_integrate
+from .manualintegrate import manualintegrate, integral_steps
+from .rationaltools import ratint
```

Comment:
The docs say that these symbols go into ```sympy.integrals``` (*not* ```sympy.integrals.integrals```!)
SymPy imports the contents of that package into its namespace; if that's what SymPy is supposed to do, then this all implies that the docs say that the symbols go into the ```sympy``` namespace (not the global namespace, I do not think we put anything into Python's global namespace).